templates/compositions/ec-component-organisation-teaser/organisation-teaser.html.twig

```diff
@@ -0,0 +1,119 @@
+{% spaceless %}
+
+{#
+/**
+ * @file
+ * Theme implementation for ECL Organisation Teaser.
+ *
+ * Available variables:
+ * - "name": name of the organisation.
+ * - "acronym": acronym of the organisation.
+ * - "logo": logo of the organisation.
+ * - "organisation_details": list of fields.
+ * - "contact_page_url": email adress or contact page to contact the organisation.
+ * - "contact_page": array with processed organisation contact page url.
+ */
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-pb-m' %}
+{% set _extra_attributes = '' %}
+{% set _address = address|default('') %}
+{% set _name = name|default('') %}
+{% set _acronym = acronym|default('') %}
+{% set _logo = logo|default({}) %}
+{% set _address = address|default([]) %}
+{% set _website = website|default([]) %}
+{% set _organisation_details = organisation_details|default([])%}
```

Comment:
_address, address, _website and website are not used, i think we should remove them.

Reply:
Done